src/store/createStore.ts

```diff
@@ -11,14 +11,14 @@ export interface Store {
   // Account
   account: {
     value: string;
-    nestedObject: {
-      nestedValue1: string;
-      nestedValue2: number;
+    keypair: {
+      publicKey: string;
+      secretKey: string;
     };
     update: (value: string) => void;
-    updateNested: (nestedVal: {
-      nestedValue1: string;
-      nestedValue2: number;
+    updateKeypair: (nestedVal: {
```

Comment:
let me know if this was intended for something else @quietbits

Reply:
No, this was just an example I used for testing. We should remove anything we don't need for the account from there.